docker.go

```diff
@@ -158,34 +158,33 @@ func (p Plugin) Exec() error {
 		os.MkdirAll(dockerHome, 0600)
 
 		path := filepath.Join(dockerHome, "config.json")
-		file, err := os.OpenFile(path, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0600)
+		err := os.WriteFile(path, []byte(p.Login.Config), 0600)
 		if err != nil {
 			return fmt.Errorf("Error writing config.json: %s", err)
 		}
-		err = os.WriteFile(path, []byte(p.Login.Config), 0600)
-		if err != nil {
-			return fmt.Errorf("Error writing config.json: %s", err)
-		}
-		file.Close()
 	}
 
 	// add base image docker credentials to the existing config file, else create new
```

Comment:
Shouldn't this comment be removed now that we're not adding credentials?

Reply:
removed it now, originally decided to keep the comments because we discussed to capture all the changed approach decisions in the comments. But it makes sense to remove it too I guess.